tests/dummy/app/routes/components/fixed-top-nav-fixed-width-body-tabs/template.hbs

```diff
@@ -0,0 +1,41 @@
+<h2>Fixed Top Nav w/ Fixed Width Body Tabs</h2>
+
+{{!- BEGIN-SNIPPET fixed-top-nav-fixed-width-body-tabs }}
+<page class="example-layout page1">
+  <hbox fit justify="between" class="blue header">
```

Comment:
Justify-between seems like a funky implimentation; while it does accomplish the intended result, it isn't explicit about our intent.

How about we axe the justify-between and swap it for an explicit wrapper around the second menu group which has an `align="right"` prop?